pyomo/solvers/tests/checks/test_gurobi.py

```diff
@@ -0,0 +1,43 @@
+import unittest
```

Comment:
Pyomo-specific suggestion: we have an extended implementation of `unittest`, so we prefer:
```
import pyomo.common.unittest as unittest
```

It still has all of the standard `unittest` libraries - just adds some extra functionality for some of our testing choices.